config/initializers/prometheus_exporter.rb

```diff
@@ -1,4 +1,4 @@
-unless Rails.env.test?
+unless Rails.env.test? || Rails.env.development?
```

Comment:
@ymao2 Great work! Just wondering, did you mean to take this out in the final PR? Wondering if you removed for testing purposes

Reply:
@Polsonby , thank you for checking my PR, yes I meant to leave it like this, this change is to switch off feature of sending performance data into prometheus_exporter server from local development, otherwise when you run app locally, you need to run prometheus_exporter as well, if not, it will keep showing some error messages which sometime are quite anonying